packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -1150,59 +1127,33 @@ contract Deploy is Deployer {
 
     /// @notice Initialize the OptimismPortal
     function initializeOptimismPortal() public broadcast {
+        console.log("Upgrading and initializing OptimismPortal proxy");
+        require(!cfg.useFaultProofs(), "Deploy: FaultProofs OptimismPortal is initialized by OPCM");
         address optimismPortalProxy = mustGetAddress("OptimismPortalProxy");
         address systemConfigProxy = mustGetAddress("SystemConfigProxy");
         address superchainConfigProxy = mustGetAddress("SuperchainConfigProxy");
-        if (cfg.useFaultProofs()) {
```

Comment:
This branch was added in #12336, but the `useFaultProofs` case is now handled by OPCM.